Microsoft.Recognizers.Text.NumberWithUnit/Portuguese/Extractors/DimensionExtractorConfiguration.cs

```diff
@@ -48,7 +48,6 @@ public class DimensionExtractorConfiguration : PortugueseNumberWithUnitExtractor
         .Concat(AreaExtractorConfiguration.AreaSuffixList)
         .Concat(LengthExtractorConfiguration.LenghtSuffixList)
         .Concat(SpeedExtractorConfiguration.SpeedSuffixList)
-        .Concat(TemperatureExtractorConfiguration.TemperatureSuffixList)
```

Comment:
Why remove this. Because portuguese did not support this? What about adding the TODO?